iset.mm

```diff
@@ -118008,6 +118042,403 @@ computer programs (as last() or lastChar()), the terminology used for
+  $}
+
+  $( The range of a prefix of a word is a subset of the set of symbols for the
+     word.  (Contributed by AV, 2-May-2020.) $)
+  pfxrn $p |- ( ( W e. Word V /\ L e. ( 0 ... ( # ` W ) ) )
+                 -> ran ( W prefix L ) C_ V ) $=
+    ( cword wcel cc0 chash cfv cfz co wa cfzo cpfx pfxf frnd ) CBDEAFCGHIJEKFAL
+    JBCAMJABCNO $.
+
+  $( A prefix consisting of at least one symbol is not empty.  (Contributed by
+     Alexander van der Vekens, 4-Aug-2018.)  (Revised by AV, 2-May-2020.) $)
+  pfxn0 $p |- ( ( W e. Word V /\ L e. NN /\ L <_ ( # ` W ) )
+                 -> ( W prefix L ) =/= (/) ) $=
+    ( cword wcel cn chash cfv cle wbr w3a cpfx co c0 wne cc0 cfzo 3ad2ant2 wceq
+    cn0 lbfzo0 ne0i sylbir wf wb cfz simp1 nnnn0 lencl 3ad2ant1 simp3 syl3anbrc
+    elfz2nn0 pfxf syl2anc f0dom0 bicomd syl necon3bid mpbird ) CBDEZAFEZACGHZIJ
+    ZKZCALMZNOPAQMZNOZVBVAVHVDVBPVGEVHAUAVGPUBUCRVEVFNVGNVEVGBVFUDZVFNSZVGNSZUE
+    VEVAAPVCUFMEZVIVAVBVDUGVEATEZVCTEZVDVLVBVAVMVDAUHRVAVBVNVDBCUIUJVAVBVDUKAVC
+    UMULABCUNUOVIVKVJVFVGBUPUQURUSUT $.
+
+  $( The value of a prefix operation for a length argument larger than the word
+     length is the empty set.  (This is due to our definition of function
+     values for out-of-domain arguments, see ~ ndmfvg ).  (Contributed by AV,
+     3-May-2020.) $)
+  pfxnd $p |- ( ( W e. Word V /\ L e. NN0 /\ ( # ` W ) < L )
+                 -> ( W prefix L ) = (/) ) $=
+    ( cword wcel cn0 chash cfv clt wbr w3a cpfx co cc0 cop csubstr c0 pfxval cz
+    wceq 3adant3 cle w3o simp1 0zd nn0z 3ad2ant2 3jca 3mix3 3ad2ant3 sylc eqtrd
+    swrdnd ) CBDZEZAFEZCGHAIJZKZCALMZCNAOPMZQUOUPUSUTTUQCAUNRUAURUONSEZASEZKNNI
+    JZANUBJZUQUCZUTQTURUOVAVBUOUPUQUDURUEUPUOVBUQAUFUGUHUQUOVEUPUQVCVDUIUJNABCU
+    MUKUL $.
+
+  $( A prefix of a word is a word over the symbols it consists of.
+     (Contributed by AV, 3-Dec-2022.) $)
+  pfxwrdsymbg $p |- ( ( S e. Word A /\ L e. NN0 )
+                     -> ( S prefix L ) e. Word ( S " ( 0 ..^ L ) ) ) $=
+    ( cword wcel wa co cc0 cfv cle wbr clt cfz simpll syl cz syl2anc syl3anc c0
+    adantl cn0 cpfx cop csubstr cfzo cima pfxval chash cuz simplr elnn0uz sylib
+    eluzfz1 simpr lencl nn0zd elfz5 mpbird swrdwrdsymbg w3o wceq 3mix3 0zd nn0z
+    wb wi adantr swrdnd mpd wrd0 eqeltrdi wo zlelttric mpjaodan eqeltrd ) BADZE
+    ZCUAEZFZBCUBGBHCUCUDGZBHCUEGUFZDZBCVPUGVSCBUHIZJKZVTWBEZWCCLKZVSWDFZVQHHCMG
+    EZCHWCMGEZWEVQVRWDNZWGCHUIIEZWHWGVRWKVQVRWDUJCUKULZHCUMOWGWIWDVSWDUNWGWKWCP
+    EZWIWDVEWLWGVQWMWJVQWCABUOUPZOCHWCUQQURABHCUSRVSWFFZVTSWBWOHHLKZCHJKZWFUTZV
+    TSVAZWFWRVSWFWPWQVBTWOVQHPECPEZWRWSVFVQVRWFNWOVCVSWTWFVRWTVQCVDTZVGHCABVHRV
+    IWAVJVKVSWTWMWDWFVLXAVQWMVRWNVGCWCVMQVNVO $.
+
+  $( The sum of the lengths of two reversed parts of a word is the length of
+     the word.  (Contributed by Alexander van der Vekens, 1-Apr-2018.)
+     (Revised by AV, 3-May-2020.) $)
+  addlenrevpfx $p |- ( ( W e. Word V /\ M e. ( 0 ... ( # ` W ) ) )
```

Comment:
~addlenrevpfx is only used in one proof (of ~lenrevpfxcctswrd), so it may be deleted (and replaced by ~addlenpfx and ~addcom in the proof of ~lenrevpfxcctswrd)

Reply:
Done.